modules/exploits/multi/browser/chrome_newfixedarray_sizecheck.rb

```diff
@@ -0,0 +1,192 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Google Chrome 83 Missing size check in NewFixedArray',
+      'Description'    => %q{
+      This module exploits an issue in Google Chrome 83.0.4103.97 (64 bit). The exploit
+      makes use of a missing size check in NewFixedArray and abuses it to create a
+      JSArray with an invalid length. This is abused to gain arbitrary read/write into
+      the isolate region. Then an ArrayBuffer can be used to achieve absolute arbitrary
```

Comment:
```suggestion
      the isolated region. Then an ArrayBuffer can be used to achieve absolute arbitrary
```

Reply:
It is not "isolated region" the v8 heap is known as isolate root.